src/DayPicker.js

```diff
@@ -92,19 +94,36 @@ const DayPicker = React.createClass({
   },
 
   render() {
-    const { month } = this.state;
+    var { month } = this.state;
```

Comment:
@majapw could you replace `var` with `let`? (here and in the following rows) thanks!